src/server/api/interacted.ts

```diff
@@ -0,0 +1,56 @@
+import { APCollection } from 'activitypub-types'
+import { Type } from '@sinclair/typebox'
+
+import type { APIConfig, FastifyTypebox } from '.'
+import type Store from '../store'
+import type ActivityPubSystem from '../apsystem.js'
+
+export const interactedRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityPubSystem) => async (server: FastifyTypebox): Promise<void> => {
```

Comment:
I don't think we need to expose this API. Is there a use case for it from sutty?

Reply:
Fair enough, deleting..